src/Elastic.CommonSchema.LoggerProvider/.config/dotnet-tools.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
We can remove this file, our build takes care of versioning and with #80 we are moving to `minver.

Reply:
Removed (it shouldn't really be in a subfolder anyway). I generally use Gitversion; I saw some comments regarding it, although I've been using it with some Github actions and it works okay..